client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.module.scss

```diff
@@ -0,0 +1,8 @@
+.root {
+  display: inherit;
+}
+
+.add-workspace-btn {
+  padding-left: 1.5em;
+  padding-right: 1.5em;
+}
```

Comment:
Thanks for this. Notice: This will be solved globally via [FP-546](https://jira.tacc.utexas.edu/browse/FP-546) concern.